operations/mimir-mixin/config.libsonnet

```diff
@@ -35,9 +35,12 @@
       overrides_exporter: 'overrides-exporter',
     },
 
+    // Custom Cluster label to use in dashboards.
+    clusterLabel: 'cluster',
```

Comment:
I don't think we should introduce this, but we should just keep using `job_labels` and `cluster_labels` everywhere. For example, `cluster_labels` is already used to build `alert_aggregation_labels` which is then used in alerts and recording rules. However, not all of them are using it so I think we should just keep working to make sure they're used everywhere (see few other comments I left).

Reply:
Will start working on using `alert_aggregation_labels`. 

But what's your idea on the approach we should take here: https://github.com/grafana/mimir/blob/ca1b6d65c4fbd84d7661e01e7c65d1f700a748e4/operations/mimir-mixin/dashboards/dashboard-utils.libsonnet#L57 for example. Because this selector should be updated accordingly.

Reply:
I see the difficulty of getting rid of this config field at all. Ok, let's keep it. Can I just ask you to rename `clusterLabel` to `per_cluster_label` to keep it consistent with `per_instance_label` and `per_node_label`, please?

The comment could be something like:
```
// The label used to differentiate between different Kubernetes clusters.
```